app/src/main/java/com/example/lhi06/residue/FindAlbumActivity.java

```diff
@@ -58,6 +59,9 @@ public void onFailure(final Call<AlbumSearchResponse> call, final Throwable t) {
 
         } catch (IOException e) {
             e.printStackTrace();
+        } catch (NullPointerException e) {
```

Comment:
okay so apprently its fine with no catch for NullPointerException now, is it something i should put in still or?